src/main/java/uk/gov/hmcts/reform/prl/services/UpdatePartyDetailsService.java

```diff
@@ -115,6 +116,15 @@ public Map<String, Object> updateApplicantRespondentAndChildData(CallbackRequest
         updatedCaseData.put(RESPONDENT_CONFIDENTIAL_DETAILS, caseDataTemp.getRespondentConfidentialDetails());
         updatedCaseData.putAll(confidentialityTabService.updateConfidentialityDetails(caseData));
 
+        String state = callbackRequest.getCaseDetails().getState();
```

Comment:
CaseUtils.getCaseData class could be used to get CaseData on line 112 and 113 as this sets the state on the CaseData